DGRunkeeperSwitch/DGRunkeeperSwitch.swift

```diff
@@ -13,7 +13,7 @@ import UIKit
 open class DGRunkeeperSwitchRoundedLayer: CALayer {
 
     override open var bounds: CGRect {
-        didSet { cornerRadius = bounds.height / 2.0 }
+		didSet { cornerRadius = self.cornerRadius }
```

Comment:
I believe that this line is no longer needed. You can remove the whole `override open var bounds: CGRect`